homeassistant/components/starlink/coordinator.py

```diff
@@ -56,6 +57,16 @@ async def _async_update_data(self) -> StarlinkData:
             except GrpcError as exc:
                 raise UpdateFailed from exc
 
+    async def stow_starlink(self, stow: bool):
+        """Set whether Starlink system tied to this coordinator should be stowed."""
+        async with async_timeout.timeout(4):
+            try:
+                await self.hass.async_add_executor_job(
+                    set_stow_state, not stow, self.channel_context
+                )
+            except GrpcError as exc:
+                raise HomeAssistantError from exc
+
     async def reboot_starlink(self):
```

Comment:
```suggestion
    async def async_stow_starlink(self, stow: bool):
        """Set whether Starlink system tied to this coordinator should be stowed."""
        async with async_timeout.timeout(4):
            try:
                await self.hass.async_add_executor_job(
                    set_stow_state, not stow, self.channel_context
                )
            except GrpcError as exc:
                raise HomeAssistantError from exc

    async def async_reboot_starlink(self):
```

We should name these `async_` since we have both sync and async function in this integration

Reply:
Makes sense, done!